partners.json

```
@@ -1051,6 +1051,26 @@
       "picture": "/images/logos/partners/epci/apgl.png",
       "height": 124,
       "width": 173
+    },
+    {
+      "name": "Grand Paris Sud - Seine - Essonne - Sénart",
+      "link": "https://data.grandparissud.fr/explore/dataset/statut-de-publication-de-la-base-d-adresse-locale-des-communes/information",
```

```suggestion
      "link": "https://data.grandparissud.fr/explore/dataset/statut-de-publication-de-la-base-d-adresse-locale-des-communes/information/",
``` 
sans le / on est rediriger sur un autre onglet